pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -664,13 +664,39 @@ func (l *LibvirtDomainManager) FinalizeVirtualMachineMigration(vmi *v1.VirtualMa
 	l.domainModifyLock.Lock()
 	defer l.domainModifyLock.Unlock()
 
+	domainName := api.VMINamespaceKeyFunc(vmi)
+	virDomain, err := l.virConn.LookupDomainByName(domainName)
+	if err != nil {
+		return err
+	}
+	defer virDomain.Free()
+
+	if err := hotPlugHostDevices(vmi, virDomain); err != nil {
+		return err
+	}
+
 	if err := l.SetGuestTime(vmi); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+// hotPlugHostDevices attach host-devices to running domain
+// Currently only SRIOV host-devices are supported
+func hotPlugHostDevices(vmi *v1.VirtualMachineInstance, domain cli.VirDomain) error {
```

Comment:
Since the parallel detach method is called `detachHostDevices` I would call this method `attachHostDevices`.

Reply:
And on other components we call it "hot-plug/unplug" (instead of attach/detach) as this is the general name of connecting/disconnecting a device from a live VM.
At virt-launcher/virtwrap scope I kept the terminology to be "attach/detach" as this is how the operation called on Libvirt

Reply:
I think @AlonaKaplan  has a point here.

> At virt-launcher/virtwrap scope I kept the terminology to be "attach/detach" as this is how the operation called on Libvirt

I guess it is a typo, as at this abstraction level you used "hotPlug".

I used "detach" at the time as this is the terms libvirt used in their API, but we can change it to hotPlug and HotUnplug.
If you think it should change, it makes sense to change it for both cases for consistency.
Consider changing the to hot* in a separate PR and keeping here that same naming.

Reply:
Sorry I meant `virt-launcher/virtwarp/device/sriov package`
We already have functions that called `attachHostDevices` on `virt-launcher/virtwarp/device/sriov` package
so we will end up with `attachHostDevices` calling `sriov.AttachHostDevices`.
I think its make more sense to keep this name, use "attach/detach" naming inside `virt-launcher/virtwarp/device/sriov `package and hotPlug/unplug outside of it.
I prefer to rename `detachHostDevices` on virt-launcher/virtwarp/manager to `hotUnplugHostDevices`
on this PR, there is no need to run all CI just for renaming one function and its still make sense to do it here
What you think?

Reply:
Fine by me.